install_config/install/prerequisites.adoc

```diff
@@ -588,6 +588,22 @@ connections, and is only required if you have clustered etcd.
 * {product-title} internal DNS cannot be received over SDN. Depending on the detected values of `*openshift_facts*`, or if the `*openshift_ip*` and `*openshift_public_ip*` values are overridden, it will be the computed value of `*openshift_ip*`. For non-cloud deployments, this will default to the IP address associated with the default route on the master host. For cloud deployments, it will default to the IP address associated with the first internal interface as defined by the cloud metadata.
 * The master host uses port *10250* to reach the nodes and does not go over SDN. It depends on the target host of the deployment and uses the computed values of `*openshift_hostname*` and `*openshift_public_hostname*`.
 
+.Aggregated Logging
+[cols='2,1,8']
+|===
+| *9200*
+|TCP
+|For Elasticsearch API use.  Required to be internally open on any infrastructure
+nodes so the Kibana component is able to retrieve logs for display.  It can be
+open externally for direct access to Elasticsearch and used with a service added
```

Comment:
externally opened for direct access to Elasticsearch by means of a route.  The route would need to be created